servant-js/src/Servant/JS/Internal.hs

```diff
@@ -87,7 +88,34 @@ defCommonGeneratorOptions = CommonGeneratorOptions
 -- Couldn't work out how to handle zero-width characters.
 --
 -- @TODO: specify better default function name, or throw error?
-toValidFunctionName :: String -> String
+toValidFunctionName :: Text -> Text
+-- @TODO: Cons text
```

Comment:
The `Cons text` TODO should be removed. it concerns a piece of code that doesn't exist anymore.

The other `TODO` as already there in the original version so I left it untouched.